### ISSUE TITLE
Up and Down change

### ISSUE THREAD

**Issue by hbellert** [Deejayfool/hass-shutter-card#83]
is it possible to rotate the up and down buttons

**Comment by abrocker** [comment]
Agree, an option to invert the Up down Buttons would be good. Invert Percentage is already active and displaying the correct state but the buttons are working the oposite direction

**Comment by epjjansen** [comment]
would like this feature too

**Comment by gabimal** [comment]
Me too! 
Please @Deejayfool help us with this change!
